happiness-backend/tests/test_reads.py

```diff
@@ -190,13 +181,35 @@ def test_get_empty_unread_happiness_3(init_client):
     assert len(happiness_list) == 0
 
 
+def test_group_unread(init_client):
```

Comment:
Would be good to add to this test where you try reading some entries and then see if they still show up in the unread entries.